hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,590 @@
+                   issued command.'''
+          resval: "0x0"
+        },
+        { bits: "29",
+          name: "TXFULL",
+          desc: '''When high, indicates that the transmit data fifo is full.
+                   Any further writes to TXD will create an error interrupt.
+                '''
+          resval: "0x0"
+        },
+        { bits: "28",
+          name: "TXEMPTY",
+          desc: '''When high, indicates that the transmit data fifo is empty.
+                   And pending commands may have to wait for more data.
+                '''
+          resval: "0x0"
+        },
+        { bits: "27"
+          name: "TXSTALL",
+          desc: '''If high, signifies that an ongoing transaction has stalled
+                   due to lack of data in the TX FIFO''',
+          resval: "0x0"
+        },
+        { bits: "26",
+          name: "TXWM",
+          desc: '''If high, the amount of data in the TX FIFO has fallen below the
+                   level of CONFIG.TX_WATERMARK words (32b each). To prevent the
+                   reassertion of this interrupt add more data ot the TXFIFO, or
+                   reduce CONFIG.TX_WATERMARK.''',
+          resval: "0x0"
+        },
+        { bits: "25"
+          name: "RXFULL",
+          desc: '''When high, indicates that the receive fifo is full.  Any
+                   ongoing transactions will stall until firmware reads some
+                   from the RXFIFO.'''
+          resval: "0x0"
+        },
+        { bits: "24"
+          name: "RXEMPTY",
+          desc: '''When high, indicates that the receive fifo is empty.
+                   Any reads from RXFIFO will cause an error interrupt.
+                   '''
+          resval: "0x0"
+        },
+        { bits: "23"
+          name: "RXSTALL",
+          desc: '''If high, signifies that an ongoing transaction has stalled
+                   due to lack of available space in the RX FIFO''',
+          resval: "0x0"
+        },
+        { bits: "20",
+          name: "RXWM",
+          desc: '''If high, the amount of data in the RX FIFO now exceeds the
+                   level of CONFIG.RX_WATERMARK words (32b each). To prevent the
+                   reassertion of this interrupt read more data from the RXFIFO, or
+                   increase CONFIG.RX_WATERMARK.''',
+          resval: "0x0"
+        },
+        { bits: "17:9",
+          name: "RXQD",
+          desc: '''Receive queue depth. Indicates how many unread entries are currently in the
+                   RX FIFO.  When active, this result may an underestimate due
+                   to synchronization delays.
+                ''',
+          resval: "0x0"
+        },
+        { bits: "8:0",
+          name: "TXQD",
+          desc: '''Transmit queue depth. Indicates how many unsent entries are currently
+                   in the TXFIFO.  When active, this result may be an
+                   over-estimate, Due to synchronization delays,
```

Comment:
I think this should be "... over-estimate due to synchronization delays."

Reply:
ok.  Will fix.